### PR TITLE
Updating these scripts to handle the change to monthly based entries

### PR DESCRIPTION
**Pull request by sitUboo**
We have a client trying to use these in ZD 78258 and they seem to be out of sync with the plugin's current format.

### CODE REVIEW

**Review by Evildethow — CHANGES_REQUESTED** [review]
`firstDayOfWeek` is also mentioned in the README - https://github.com/cloudbees/user-activity-monitoring-scripts/blob/1133d80b60ec720dec1716b1db1595c08c653da3/README.md

File: README.md [hunk 1]
```diff
@@ -13,23 +13,23 @@ Detailed activity reports can be generated across multiple masters using the fol
...
         "name": "simon"
     }
     ```
 * [count-user-activity.sh](./count-user-activity.sh)
     * displays on the stdout the summary of the users counted from `/out/aggregated-user-activity.json`:
     ```
-    1 user(s) counted as Authenticated from 2018-10-28 to 2019-02-03
-    3 user(s) counted as SCM contributor from 2018-10-28 to 2018-12-30
-    4 user(s) counted as CloudBees Core users from 2018-10-28 to 2019-02-03
+    1 user(s) counted as Authenticated from 2018-10-01 to 2019-02-01
```
Comment (alecharp) [inline README.md]: Shouldn't the end date be `2018-31-12` if you count by months? Or maybe I misunderstood this all month thing.

**Review by alecharp — COMMENTED** [review]
Might need some explanation about when it start and when it ends.

File: README.md (hunk 1 above)
Comment (sitUboo) [reply README.md]: I updated the readme with the current output (at least against the instance I tested this against).

**Pull request merged**